src/main/java/com/thevoxelbox/voxelsniper/brush/type/stencil/StencilBrush.java

```diff
@@ -35,10 +34,11 @@
  */
 public class StencilBrush extends AbstractBrush {
```

Comment:
Using a pipe as delimiter for words ending with an `l` are quite hard to read imo

Also, how does one use this tool? I currently have no stencil in place, so I used `/b sl ffs`, Favs tells me I'm missing arguments. According to info parameter, this is the correct syntax. If the file doesn't exist, Favs should let the actor know about it, if that's the case here.

Reply:
> Using a pipe as delimiter for words ending with an l are quite hard to read imo
Oof, whole format should be changed everywhere if changed here no?

Ty, you found a bug here.

Reply:
> Oof, whole format should be changed everywhere if changed here no?

For pipes?

Reply:
yes

Reply:
I only spotted this one yet. Other cases may be fine?

Reply:
Right, other cases may be fine, ``l|`` is only used in Stencil brushes.

Reply:
👍🏻